ethcore/src/miner/mod.rs

```diff
@@ -133,6 +133,9 @@ pub trait MinerService : Send + Sync {
 	/// On chains where sealing is done externally (e.g. PoW) we provide only reward beneficiary.
 	fn set_author(&self, author: Author);
 
+	/// Clears the engine signer and stops signing.
+	fn clear_author(&self);
```

Comment:
Do you think the same "trick" would work here too? I.e. avoid adding a new trait method and instead change `set_author` to take an `Option<Author>` instead?

Reply:
Done.
I made it a `T: Into<Option<Author>>`, so that you can pass in just an `Author`, and we don't have to update all the call sites.
Also, I added `*self.signer.write() = None` to `MinerService::set_author`, which seemed like an omission. Or am I misreading this?

Reply:
(And: Greetings from Solna! :slightly_smiling_face: :sweden:)